src/main/scala/org/broadinstitute/dsde/vault/services/uBAMCollection/UBamCollectionDescribeService.scala

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.dsde.vault.services.uBAMCollection
+
+import akka.actor.Props
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.vault.DmClientService
+import org.broadinstitute.dsde.vault.model._
+import org.broadinstitute.dsde.vault.services.VaultDirectives
+import org.broadinstitute.dsde.vault.common.directives.VersioningDirectives._
+import spray.http.MediaTypes._
+import spray.routing._
+
+@Api(value = "/ubamcollections", description = "uBamCollection Service", produces = "application/json")
+trait UBamCollectionDescribeService extends HttpService with VaultDirectives {
+
+  private final val ApiVersions = "v1"
+
+  val ubcdRoute = ubamCollectionDescribeRoute
+
+  @ApiOperation(value = "Describes an uBamCollection",
+    nickname = "uBamCollection_describe",
+    httpMethod = "GET",
+    produces = "application/json",
+    response = classOf[UBamCollection]
+  )
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "version", required = true, dataType = "string", paramType = "path", value = "API version", allowableValues = ApiVersions),
+    new ApiImplicitParam(name = "id", required = true, dataType = "string", paramType = "path", value = "uBamCollection Vault ID")
+  ))
+  @ApiResponses(Array(
+    new ApiResponse(code = 200, message = "Successful Request"),
+    new ApiResponse(code = 404, message = "Vault ID Not Found"),
+    new ApiResponse(code = 500, message = "Vault Internal Error")
+  ))
+  def ubamCollectionDescribeRoute = {
+    pathVersion("ubamcollections", Segment) { (versionOpt, id) =>
```

Comment:
I'm bouncing around several PRs at the moment, so sorry if I missed this before.

Please don't use `pathVersion()` on any _new_ routes. That's a backwards compatible "hack", only for old routes that didn't version from the beginning.

Use `path("ubamscollections" / v ~ IntNumber / Segment)`.